programs/uxd/src/events.rs

```diff
@@ -109,6 +109,19 @@ pub struct SetDepositoryProfitsBeneficiaryCollateralEvent {
     pub profits_beneficiary_collateral: Pubkey,
 }
 
+/// Event called in [instructions::edit_*_depository::handler].
+#[event]
+pub struct SetDepositoryRebalancingTargetWeightEvent {
```

Comment:
I don't think it's rebalancing targets, but simply targetWeight (could be then approached through rebalancing, minting, redeeming etc)